### PR TITLE
Fix new clippy lints

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
I'll need to send a PR to gir too to fix the missing `#[must_use]` in the auto code.

### CODE REVIEW

File: cairo/src/device.rs [hunk 1]
```diff
@@ -157,7 +157,7 @@ impl Device {
             let status = ffi::cairo_device_acquire(self.to_raw_none());
             status_to_result(status)?;
         }
-        Ok(DeviceAcquireGuard { 0: self })
```
Comment (sdroege) [inline cairo/src/device.rs:160]: I didn't consider that this was valid syntax :rofl:

File: gio/src/subclass/action_map.rs [hunk 2]
```diff
@@ -101,7 +101,7 @@ unsafe extern "C" fn action_map_lookup_action<T: ActionMapImpl>(
 
         let mut map = wrap
             .steal_qdata::<HashMap<String, Action>>(*ACTION_MAP_LOOKUP_ACTION_QUARK)
-            .unwrap_or_else(HashMap::new);
+            .unwrap_or_else_default();
```
Comment (sdroege) [inline gio/src/subclass/action_map.rs:104]:
```suggestion
            .unwrap_or_default();
```

File: cairo/src/device.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply cairo/src/device.rs:160]: It's very convenient when writing macros. :3

**Comment by sdroege** [comment]
Looks good otherwise, thanks!

**Comment by GuillaumeGomez** [comment]
Once the gir PR is in, I'll send PRs to all repositories (including gstreamer ;) ).

**Review by sdroege — APPROVED** [review]

**Pull request merged**